stories/shared-components/checkbox/Checkbox.stories.tsx

```diff
@@ -72,30 +96,30 @@ LabelledCheckboxStory.storyName = 'with label';
 export const LongLabelledCheckboxStory = (args: DefaultArgs) => (
```

Comment:
Need a storyName field on this story, similar to other stories. Otherwise Storybook uses the name of the function for the story name:

<img src="https://user-images.githubusercontent.com/6834224/151396812-69395dd0-64ce-4e6b-80fb-cdc902d591e8.png" width="300" />

Reply:
✅